src/components/Proposals/ProposalPage/OPProposalPage/ProposalVotesSummaryDetails/ProposalVotesSummaryDetails.tsx

```diff
@@ -0,0 +1,203 @@
+import Image from "next/image";
+import { HStack, VStack } from "@/components/Layout/Stack";
+
+import infoTransparentIcon from "@/icons/info-transparent.svg";
+import checkIcon from "@/icons/check.svg";
+import linkIcon from "@/icons/link.svg";
+import ProposalVotesBar from "../ProposalVotesBar/ProposalVotesBar";
+import { Proposal } from "@/app/api/common/proposals/proposal";
+import TokenAmountDisplay from "@/components/shared/TokenAmountDisplay";
+import { ParsedProposalResults } from "@/lib/proposalUtils";
+import { format } from "date-fns";
+import Link from "next/link";
+import { formatNumber } from "@/lib/utils";
+
+import Tenant from "@/lib/tenant/tenant";
+const { token } = Tenant.current();
+
+function formatNumberWithScientificNotation(x: any) {
```

Comment:
This exact method is used in the VotingTimelineChart. What do you think about moving them out in a shared helper method?